Source/WebCore/Modules/compression/CompressionStreamEncoder.h

```diff
@@ -73,6 +83,14 @@ class CompressionStreamEncoder : public RefCounted<CompressionStreamEncoder> {
     bool m_didFinish { false };
     z_stream m_zstream;
 
+    bool m_usingAppleCompressionFramework { false };
```

Comment:
This seems like it could be better organized by having something like a std::optional<CompressionStreamWrapper> instead.